components/concept/engine/src/main/java/mozilla/components/concept/engine/search/SearchAdapter.kt

```diff
@@ -0,0 +1,21 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.concept.engine.search
+
+/**
+ * May be implemented by client code in order to allow a component to start searches.
+ */
+interface SearchAdapter {
```

Comment:
I am not convinced that this should be in `concept-engine`. It's not really related to the browser engine. Can this just live in feature-search?

Reply:
It's used by the `DefaultSelectionActionDelegate` in feature-contextmenu and `BrowserStoreSearchAdapter` in feature-search.  It's possible that there's a more recent ancestor of the two though, I'm not super familiar with the component structure.

Reply:
@Baron-Severin yeah there's no common ancestor, but `feature-contextmenu` can depend on `feature-search`. It already depends on `feature-tabs` and `feature-app-links`

Reply:
That makes a lot of sense.  Moving it.